.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "test/cbmc/aws-templates-for-cbmc-proofs"]
 	path = test/cbmc/aws-templates-for-cbmc-proofs
 	url = https://github.com/awslabs/aws-templates-for-cbmc-proofs.git
+[submodule "test/cbmc/litani"]
```

Comment:
I recommend that this be submoduled into the tools directory parallel to the template starter kit.

Reply:
The template starter kit currently resides under `test/cbmc`, not under the tools directory. So I did put litani parallel to the starter kit. I think you're thinking of the old repo?

Reply:
Or are you suggesting that I move both of the submodules to he `tools` directory in this PR?

Reply:
No, it should be parallel to the starter kit, so you might be doing the right thing.